onnxruntime/test/ir/onnx_model_test.cc

```diff
@@ -76,7 +76,6 @@ TEST_F(ONNXModelsTest, future_opset) {
   std::shared_ptr<Model> model;
   common::Status st = Model::Load(ORT_TSTR("./testdata/add_opset_314159.onnx"), model, nullptr, *logger_);
   ASSERT_FALSE(st.IsOK());
-  ASSERT_EQ(st.Code(), common::INVALID_GRAPH);
```

Comment:
Can we keep this?

Reply:
Now this is caught during Model creation and therefore the execution never reaches graph resolve where this used to be caught earlier.